### ISSUE TITLE
Open-Sourcing this app

### ISSUE THREAD

**Issue by winsmith** [TelemetryDeck/TelemetryViewer#87]
It turns out that, because we are just a few people, we cannot maintain a good native app and a good web app for TelemetryDeck right now. However, the web app is the more universal choice and it allows us to reach more customers and is easier to start with. This means that right now, the web app at https://dashboard.telemetrydeck.com must become the primary app for using TelemetryDeck.

However, we don't want to completely shut down the iOS and macOS apps. Instead, we're open-sourcing them to our awesome community. This way, I won't feel so bad if SwiftUI bugs are unfixed for weeks on end 😅. And it has the added benefit that our customers can create their own custom forks that offer functionality just for them. 

This ticket is here to collect all the todos that need to happen before we can switch the repo from "private" to "public".

- [x] Check that all dependencies (especially the models package) are open source
- [x] Check if any other secrets are in the repo 
- [x] Define a license
- [x] Move UIComponents inside TelemetryViewer repo
- [x] Move SwiftUICharts inside TelemetryViewer repo
- [x] Make main branch protected
- [x] Remove all pricing information and refer to the web instead
- [x] #92 
- [x] Write README
- [x] #131

The build and release process is not part of this ticket -- for now we'll continue using our existing manual labor process and investigate fastlane in the future.  

## Fastlane

In the future, we want to make sure this app is build and released using fastlane. Here's a ticket for that: #89 

## Related Todos that belong to other repos

- [x] Add README for DataTransferObjects repo
- [x] Add README for SwiftDateOperations repo
- [x] Delete APIClient repo
- [x] Delete UIComponents repo
- [x] Move over issues from the "Viewer" Repo
- [ ] Host the releases in this repo instead "Viewer"
- [ ] Rename the Viewer Repo to "issues" in order for it to become the central issue reporting place
